sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultCertificatesInfo.java

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.jca;
+
+import java.security.Key;
+import java.security.cert.Certificate;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Date;
+import java.util.Collections;
+import java.util.Objects;
+
+/**
+ * Store certificates in portal
```

Comment:
what is KeyVaultCertificatesInfo used for?
If it is more like an util class, can we make it a static class ?

Reply:
Each KeyVaultKeyStore has it's own KeyVaultCertificatesInfo, the function of KeyVaultCertificatesInfo is just to make sure when get properties about keyvault, it will get latest info if user setted refresh time.